app/Enums/CategoryEnum.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Enums;
+
+enum CategoryEnum: int
+{
+    case CLOTHES = 1;
+    case ELECTRONICS = 2;
+    case OTHER = 3;
+
+    public function name(): string
+    {
+        return match($this) {
+            self::CLOTHES => 'clothes',
+            self::ELECTRONICS => 'electronics',
+            self::OTHER => 'other',
+        };
+    }
+
+    public static function getCategoriesIds()
```

Comment:
Nie lepiej zrobić jakąś klasę abstrakcyjną do tego i dać jakąś inukatową nazwę aby dla każdego enuma była ta sama?

Reply:
myślałem o tym, ale jak dodam jakąś generyczną nazwę do tego to nie będzie to później zbyt niejasne jak będę z tego korzystał?